primitives/consensus/aura/src/inherent_digest.rs

```diff
@@ -0,0 +1,33 @@
+#[cfg(feature = "std")]
```

Comment:
This is not a new code. Moved from: https://github.com/input-output-hk/polkadot-sdk/blob/partnerchains-stable2407/substrate/primitives/inherents/src/lib.rs#L401